src/workloads/serverless/IndexStress.yml

```diff
@@ -0,0 +1,269 @@
+SchemaVersion: 2018-07-01
+Owner: Atlas Serverless
```

Comment:
Is there an @-mentionable github username that could be put here for owner?

Reply:
As far as I know, Atlas teams don't have team github usernames like the server teams. I saw a few examples of workloads where the team name was just written out (like [this one](https://github.com/mongodb/genny/blob/0c154851e154663b5cdbb804f001019f7901bd7a/src/workloads/scale/ScanWithLongLived.yml#L2)), so I thought that would be appropriate here. @JamesHeppenstall , can you confirm that we don't have a team username like this?

Reply:
Yeah afaik the teams in the `10gen` org are being used to determine permissions and not used for anything else. Also, not sure if we'd even be able to reference a team from another GitHub org here.

Reply:
Gotcha fair, just wanted to check!